project-codeflare/codeflare-sdk
# remove: DDPJobDefinition from SDK

# Issue link
https://issues.redhat.com/browse/RHOAIENG-5096

# What changes have been made
This pr contains a removal of the  DDPJobDefinition from the codeflare-sdk, as well as updated tests to reflect the change. 

A follow up pr will be made to update the docs and demo notebooks upon being merged.

# Verification steps
Run the sdk tests and ensure they are functioning correctly. 
Run a demo notebook to ensure no breaks.

## Checks
- Testing Strategy
